src/Files/File.php

```diff
@@ -776,11 +776,7 @@ public function addFixableError(
         $severity=0
     ) {
         $recorded = $this->addError($error, $stackPtr, $code, $data, $severity, true);
-        if ($recorded === true && $this->fixer->enabled === true) {
-            return true;
-        }
-
-        return false;
+        return $recorded === true && $this->fixer->enabled === true;
```

Comment:
This isn't how I write code. I prefer to use the IF statement for my own readability.